crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -469,6 +470,11 @@ en:
         raid10: 'RAID 10'
       raid_disks_selected: 'Failed to save %{node}: Not enough disks selected for this RAID level'
 
+  upgrade:
+    upgrade:
+      nodes_not_ready: 'Some nodes are not ready: %{nodes}.
+Fix the state of nodes before proceeding with the upgrade.'
```

Comment:
Can you pick one of the answer at http://stackoverflow.com/questions/10982706/how-to-break-lines-in-yml-localization-files-on-rails for the new line?